src/dashboard/src/components/administration/forms.py

```diff
@@ -281,7 +281,7 @@ class ProcessingConfigurationForm(forms.Form):
 
     def __init__(self, *args, **kwargs):
         super(ProcessingConfigurationForm, self).__init__(*args, **kwargs)
-        self._load_processing_config_fields()
+        self._load_processing_config_fields(kwargs.pop("user"))
```

Comment:
I am not sure why init isn't accepting user in the `**kwargs`, but it seems that these two lines work when switched to pop user first:

```
        self._load_processing_config_fields(kwargs.pop("user"))
        super(ProcessingConfigurationForm, self).__init__(*args, **kwargs)
```

We see this otherwise:
```
archivematica-dashboard_1_a00bb1a4162c | ERROR     2018-12-21 13:25:07  archivematica.dashboard:views_processing:_report_error:54:  Unable to load processing configuration page. __init__() got an unexpected keyword argument 'user'
archivematica-dashboard_1_a00bb1a4162c | Traceback (most recent call last):
archivematica-dashboard_1_a00bb1a4162c |   File "/src/dashboard/src/components/administration/views_processing.py", line 67, in edit
archivematica-dashboard_1_a00bb1a4162c |     request.POST or None, user=request.user)
archivematica-dashboard_1_a00bb1a4162c |   File "/src/dashboard/src/components/administration/forms.py", line 283, in __init__
archivematica-dashboard_1_a00bb1a4162c |     super(ProcessingConfigurationForm, self).__init__(*args, **kwargs)
archivematica-dashboard_1_a00bb1a4162c | TypeError: __init__() got an unexpected keyword argument 'user'
````

Reply:
Thank you! Fixed now. The base class must be defining the keywords explicitly.

Reply:
I wonder why I can't mark this as resolved... maybe because it's a single comment and not a review?

Hey btw, we should try the "Suggested Changes" feature at some point!

Reply:
Ahh! Looks like maybe only the author can! 

Will give that feature a try too next time. I can't keep up with Gh these days!!